lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -160,14 +168,22 @@ def upload(app_id, binary_path, platform)
         upload_status_response.release_id
       end
 
-      # Gets the upload status for the app release.
-      def get_upload_status(app_id, app_token)
+      # Gets the upload_status for this
+      #
+      # args
+      #   app_id - Firebase App ID
+      #   upload_token - URL encoded upload token
+      #
+      # Returns the release_id on a successful release, otherwise returns nil.
```

Comment:
For documentation like this, I think referring to it in sentence case is preferred unless the identifier is meaningful somehow: "release ID"

Reply:
I see you resolved this, but did you forget to push up a commit?

Reply:
Yup I did, pushed the new commit now